release/models/bgp/openconfig-bgp-common.yang

```diff
@@ -698,17 +704,52 @@ submodule openconfig-bgp-common {
     description
       "Parameters controlling usage of  of DMZ Link-Bandwidth
       extended community in pultipath RIB/FIB formation";
+
     leaf enabled {
       type boolean;
       description
-      "When set to TRUE, BGP multiplepath shall distributed traffic
-      load among contributing routes proportionally to value of
-      Local Administrator subfield of link-bandwidth extended
-      community [draft-ietf-idr-link-bandwidth-07].
-      This leaf has no effect if BGP multi-path is disabled or
-      if maximum-path attribute of BGP multi-path value is set
-      to 1";
+        "When set to TRUE, BGP multiplepath shall distributed traffic
+        load among contributing routes proportionally to value of
+        Local Administrator subfield of link-bandwidth extended
+        community [draft-ietf-idr-link-bandwidth-07].
+        This leaf has no effect if BGP multi-path is disabled or
+        if maximum-path attribute of BGP multi-path value is set
+        to 1";
+    }
+
+    leaf cumulative {
+      type boolean;
+      when "../enabled = 'TRUE'";
+      description
+        "When set to TRUE, and sending of link-bandwidth community values
+        is enabled, the system will sum the bgp link-bandwidth community
+        values for equal cost paths as defined in draft-ietf-bess-ebgp-dmz.";
+      reference
+        "https://datatracker.ietf.org/doc/draft-ietf-bess-ebgp-dmz/";
+    }
+
+    leaf non-transitive {
```

Comment:
I'll add a general comment to the PR, but expect an update to the base link-bw I-D where both transitive and non-transitive forms are dealt with.  This flag will be inconsistent with upcoming procedure.

The desired behavior is "send non-transitive link-bw extended communities over an ebgp session", and the name needs more context for that.  

Perhaps "send-non-transitive-via-ebgp" ?

Reply:
I elaborated on the description.  I would rather keep the leaf name somewhat short.  With the updated description do you think the leaf name `non-transitive-ebgp` is good enough?